runtime/README.md

```diff
@@ -118,6 +118,26 @@ Table: `CALIPTRA_FW_LOAD` output arguments
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips_status | u32      | Indicates if the command is FIPS approved or an error
 
+### FW\_INFO
```

Comment:
Is there a function for returning more details firmware versions? The ROM can provide its git commit and image hash...

Reply:
This just documents what exists today. If we want to add more things to this response that's fine (in a follow-up PR)